### DISCUSSION TITLE
context menu and node right click

### DISCUSSION THREAD

**Discussion by pfh59** [Blazor-Diagrams/Blazor.Diagrams#318] [Q&A]
Hi, 

How can i disable right click on diagram or node?
I'm using context menu, but if we right click on node, node still selected and movable even if your mouse button are up...

we can reproduce this behavior in the demo easily

I remain at your disposal for more information

**Comment by zHaytam** [comment]
Hey, please tell me how you reproduce the issue

**Reply by pfh59** [reply]
Hi,

Sorry for my late reply,

In attachement a video to describe right click issue.
Environnement : 

- osx : ventura 13.4
- Safari : 16.5


1. Left click stay pressed :
  - moving mouse
  - node move : 
  => no problem 🎉 ✅ 
3. Right click pressed  or stay pressed:
- Browser popup appear 😵‍💫 
- Left click to remove popup and the focus stay on node
- Move your move without any button pressed
=> Node still moving without any mouse buttons pressed... 🚫 

https://github.com/Blazor-Diagrams/Blazor.Diagrams/assets/109206285/837f11cb-9611-4176-9a7f-d5e684a820ac

I find a workaround, I remove DragMoveBehavior and add my owned DragMoveBehavior

**Reply by zHaytam** [reply]
I see now. Could you please create an issue about this? Can you also mention what you changed in DragMoveBehavior to fix this?

I am assuming you only move nodes on left click instead of all clicks, please correct me if otherwise.

**Reply by pfh59** [reply]
hi, issue is created : #329
